components/Navbar.tsx

```diff
@@ -145,6 +153,16 @@ const Navbar = () => {
                 >
                   Profile Settings
                 </li>
+                {!userStore.user?.hasCompletedKyc && (
+                  <Link href="/kyc" passHref>
+                    <li
```

Comment:
<a> tag here as well. What putting an <a> tag there does is that it generates a link to "right click" open new tab